src/enums/settlements.js

```diff
@@ -47,7 +48,9 @@ const ValidSettlementModels = [
   { settlementDelay: 'IMMEDIATE', settlementGranularity: 'GROSS', settlementInterchange: 'BILATERAL' },
   { settlementDelay: 'DEFERRED', settlementGranularity: 'GROSS', settlementInterchange: 'BILATERAL' },
   { settlementDelay: 'DEFERRED', settlementGranularity: 'NET', settlementInterchange: 'BILATERAL' },
-  { settlementDelay: 'DEFERRED', settlementGranularity: 'NET', settlementInterchange: 'MULTILATERAL' }
+  { settlementDelay: 'DEFERRED', settlementGranularity: 'NET', settlementInterchange: 'MULTILATERAL' },
+  { settlementDelay: 'CONTINUAL', settlementGranularity: 'GROSS', settlementInterchange: 'BILATERAL' }
```

Comment:
It looks like the option is already present for CGS i.e. IMMEDIATE, GROSS, BILATERAL so this change probably isn't needed

Reply:
if that is the case then the story is already done and functionality available in central-ledger.?